### ISSUE TITLE
🔬 SciML — 2026-04-02 (41 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#25]
### 🏆 Oral

**1. [RealPDEBench: A Benchmark for Complex Physical Systems with Real-World Data](https://openreview.net/forum?id=y3oHMcoItR)**
  *Peiyan Hu, Haodong Feng, Hongyuan Liu et al.*
  📍 ICLR 2026
  > Predicting the evolution of complex physical systems remains a central problem in science and engineering. Despite rapid progress in scientific Machine Learning (ML) models, a critical bottleneck i...

**2. [The dark side of the forces: assessing non-conservative force models for atomistic machine learning](https://openreview.net/forum?id=OEl3L8osas)**
  *Filippo Bigi, Marcel F. Langer, Michele Ceriotti*
  📍 ICML 2025
  > The use of machine learning to estimate the energy of a group of atoms, and the forces that drive them to more stable configurations, have revolutionized the fields of computational chemistry and m...

**3. [It's All Just Vectorization: einx, a Universal Notation for Tensor Operations](https://openreview.net/forum?id=QqvQ3iAdpC)**
  *Florian Fervers, Sebastian Bullinger, Christoph Bodensteiner et al.*
  📍 ICLR 2026
  > Tensor operations represent a cornerstone of modern scientific computing. However, the Numpy-like notation adopted by predominant tensor frameworks is often difficult to read and write and prone to...

### ⭐ Spotlight

**4. [Accelerating Visual-Policy Learning through Parallel Differentiable Simulation](https://openreview.net/forum?id=4frj038M6W)**
  *Haoxiang You, Yilang Liu, Ian Abraham*
  📍 NeurIPS 2025
  > In this work, we propose a computationally efficient algorithm for visual policy learning that leverages differentiable simulation and first-order analytical policy gradients.
Our approach decouple...

**5. [On the Guidance of Flow Matching](https://openreview.net/forum?id=pKaNgFzJBy)**
  *Ruiqi Feng, Chenglei Yu, Wenhao Deng et al.*
  📍 ICML 2025
  > Flow matching has shown state-of-the-art performance in various generative tasks, ranging from image generation to decision-making, where generation under energy guidance (abbreviated as guidance i...

**6. [Learning Boltzmann Generators via Constrained Mass Transport](https://openreview.net/forum?id=KCIbELeuK0)**
  *Christopher von Klitzing, Denis Blessing, Henrik Schopmans et al.*
  📍 NeurIPS2025-AI4Science Spotlight
  > Efficient sampling from high-dimensional and multimodal unnormalized probability distributions is a central challenge in many areas of science and machine learning. We focus on Boltzmann generators...

**7. [FlashMD: long-stride, universal prediction of molecular dynamics](https://openreview.net/forum?id=ogZu06NgQs)**
  *Filippo Bigi, Sanggyu Chong, Agustinus Kristiadi et al.*
  📍 NeurIPS 2025
  > Molecular dynamics (MD) provides insights into atomic-scale processes by integrating over time the equations that describe the motion of atoms under the action of interatomic forces. Machine learni...

**8. [Is Grokking a Computational Glass Relaxation?](https://openreview.net/forum?id=Tk5nQnTGmP)**
  *Xiaotian Zhang, Yue Shang, Entao Yang et al.*
  📍 NeurIPS 2025
  > Understanding neural network' (NN) generalizability remains a central question in deep learning research.
The special phenomenon of grokking, where NNs abruptly generalize long after the training p...

**9. [ELECTRA: A Cartesian Network for 3D Charge Density Prediction with Floating Orbitals](https://openreview.net/forum?id=Rz9ISbLxf0)**
  *Jonas Elsborg, Luca Thiede, Alan Aspuru-Guzik et al.*
  📍 NeurIPS 2025
  > We present the Electronic Tensor Reconstruction Algorithm (ELECTRA) - an equivariant model for predicting electronic charge densities using floating orbitals. Floating orbitals are a long-standing ...

**10. [Decomposing Interventional Causality into Synergistic, Redundant, and Unique Components](https://openreview.net/forum?id=yPnEvPq3kV)**
  *Abel Jansma*
  📍 NeurIPS 2025
  > We introduce a novel framework for decomposing interventional causal effects into synergistic, redundant, and unique components, building on the intuition of Partial Information Decomposition (PID)...

**11. [Cloud4D: Estimating Cloud Properties at a High Spatial and Temporal Resolution](https://openreview.net/forum?id=g2AAvmBwkS)**
  *Jacob Lin, Edward Gryspeerdt, Ronald Clark*
  📍 NeurIPS 2025
  > There has been great progress in improving numerical weather prediction and climate models using machine learning. However, most global models act at a kilometer-scale, making it challenging to mod...

**12. [Learning Dynamics under Environmental Constraints via Measurement-Induced Bundle Structures](https://openreview.net/forum?id=mwSBIlNLdQ)**
  *Dongzhe Zheng, Wenjie Mei*
  📍 ICML 2025
  > Learning unknown dynamics under environmental (or external) constraints is fundamental to many fields (e.g., modern robotics), particularly challenging when constraint information is only locally a...

**13. [Parallel Simulation for Log-concave Sampling and Score-based Diffusion Models](https://openreview.net/forum?id=qtuxDy2qEB)**
  *Huanjian Zhou, Masashi Sugiyama*
  📍 ICML 2025
  > Sampling from high-dimensional probability distributions is fundamental in machine learning and statistics. As datasets grow larger, computational efficiency becomes increasingly important, particu...

### 📌 Poster

**14. [Common Task Framework For a Critical Evaluation of Scientific Machine Learning Algorithms](https://openreview.net/forum?id=pkFKjmUE3L)**
  *Philippe Martin Wyder, Judah A Goldfeder, Alexey Yermakov et al.*
  📍 NeurIPS 2025 Datasets and Benchmarks Track poster
  > Machine learning (ML) is transforming modeling and control in the physical, engineering, and biological sciences. However, rapid development has outpaced the creation of standardized, objective ben...

**15. [Towards Foundation Models for Scientific Machine Learning: Characterizing Scaling and Transfer Behavior](https://openreview.net/forum?id=zANxvzflMl)**
  *Shashank Subramanian, Peter Harrington, Kurt Keutzer et al.*
  📍 NeurIPS 2023 poster
  > Pre-trained machine learning (ML) models have shown great performance for a
wide range of applications, in particular in natural language processing (NLP)
and computer vision (CV). Here, we study h...

**16. [Scientific Machine Learning for Symbolic Recovery of Relativistic Effects in Black Hole Orbits](https://openreview.net/forum?id=pcUoDfa4Yd)**
  *Pothuraju Naveen Yadav, Prathamesh Dinesh Joshi, Raj Dandekar et al.*
  📍 NeurIPS2025-AI4Science Poster
  > Simulating relativistic orbital dynamics around Schwarzschild black holes is essential for understanding general relativity and astrophysical phenomena like precession. Traditional numerical solver...

**17. [The Darwin–Gödel Discovery Machine: Toward Bounded-Risk Self-Improving AI4Science](https://openreview.net/forum?id=rgAIYyXzww)**
  *Xuening Wu, Xinhang Zhang, Yanlan Kang et al.*
  📍 NeurIPS2025-AI4Science Poster
  > We present the Darwin–Gödel Discovery Machine (DGDM), a dual-loop system for risk-aware self-improving AI4Science. The inner Darwinian loop evolves candidate solutions—demonstrated here with molecu...

**18. [Differentiable Simulation of Hard Contacts with Soft Gradients for Learning and Control](https://openreview.net/forum?id=2EGtfFwxx8)**
  *Anselm Paulus, Andreas René Geist, Pierre Schumacher et al.*
  📍 ICLR 2026
  > Contact forces introduce discontinuities into robot dynamics that severely limit the use of simulators for gradient-based optimization. Penalty-based simulators such as MuJoCo, soften contact resol...

**19. [Accelerated Learning with Linear Temporal Logic using Differentiable Simulation](https://openreview.net/forum?id=zbdhhlIy8o)**
  *Alper Kamil Bozkurt, Calin Belta, Ming Lin*
  📍 ICLR 2026
  > Ensuring that reinforcement learning (RL) controllers satisfy safety and reliability constraints in real-world settings remains challenging: state-avoidance and constrained Markov decision processe...

**20. [Context parroting: A simple but tough-to-beat baseline for foundation models in scientific machine learning](https://openreview.net/forum?id=EUAXc9Hlvm)**
  *Yuanzhao Zhang, William Gilpin*
  📍 ICLR 2026
  > Recent time-series foundation models exhibit strong abilities to predict physical systems. These abilities include zero-shot forecasting, in which a model forecasts future states of a system given ...

**21. [AI for Open Science: A Multi-Agent Perspective for Ethically Translating Data to Knowledge](https://openreview.net/forum?id=vhEUYsNrGa)**
  *Chase Yakaboski, Gregory Hyde, Clement Nyanhongo et al.*
  📍 NeurIPS2023-AI4Science Poster
  > AI for Science (AI4Science), particularly in the form of self-driving labs, has the potential to sideline human involvement and hinder scientific discovery within the broader community. While prior...

**22. [Spectrum-Informed Multistage Neural Network: Multiscale Function Approximator of Machine Precision](https://openreview.net/forum?id=59hdCpJyHx)**
  *Jakin Ng, Yongji Wang, Ching-Yao Lai*
  📍 ICML2024-AI4Science Poster
  > Deep learning frameworks have become powerful tools for approaching scientific problems such as turbulent flow, which has wide-ranging applications. In practice, however, existing scientific machin...

**23. [PDEfuncta: Spectrally-Aware Neural Representation for PDE Solution Modeling](https://openreview.net/forum?id=NfBrMDF0Xi)**
  *Minju Jo, Woojin Cho, Uvini Balasuriya Mudiyanselage et al.*
  📍 NeurIPS 2025 poster
  > Scientific machine learning often involves representing complex solution fields that exhibit high-frequency features such as sharp transitions, fine-scale oscillations, and localized structures. Wh...

### 🆕 Recent (ArXiv / OpenReview)

**24. [Scientific Machine Learning Benchmarks](https://openreview.net/forum?id=J4Hy8UnpPl)**
  *Jeyan Thiyagalingam, Mallikarjun Shankar, Geoffrey C. Fox et al.*
  📍 CoRR 2021
  > The breakthrough in Deep Learning neural networks has transformed the use of AI and machine learning technologies for the analysis of very large experimental datasets. These datasets are typically ...

**25. [Mechanistic Neural Networks for Scientific Machine Learning](https://openreview.net/forum?id=6TR7dg23YF)**
  *Adeel Pervez, Francesco Locatello, Efstratios Gavves*
  📍 CoRR 2024
  > This paper presents Mechanistic Neural Networks, a neural network design for machine learning applications in the sciences. It incorporates a new Mechanistic Block in standard architectures to expl...

**26. [Verification and Validation for Trustworthy Scientific Machine Learning](https://openreview.net/forum?id=TdsfHEhzuw)**
  *John D. Jakeman, Lorena A. Barba, Joaquim R. R. A. Martins et al.*
  📍 CoRR 2025
  > Scientific machine learning (SciML) models are transforming many scientific disciplines. However, the development of good modeling practices to increase the trustworthiness of SciML has lagged behi...

**27. [Overview of ACM SIGKDD 2024 AI4Science4AI Special Day](https://openreview.net/forum?id=3gbzJPV1qe)**
  *Wei Ding, Gustau Camps-Valls*
  📍 KDD 2024
  > This paper provides an overview of the ACM SIGKDD 2024 AI4Science4AI special day. It includes information about the organizers, invited speakers, keynote speakers, the event agenda, and insights fr...

**28. [Bridging AI and Science: Implications from a Large-Scale Literature Analysis of AI4Science](https://openreview.net/forum?id=uDSIPHgDvI)**
  *Yutong Xie, Yijun Pan, Hua Xu et al.*
  📍 CoRR 2024
  > Artificial Intelligence has proven to be a transformative tool for advancing scientific research across a wide range of disciplines. However, a significant gap still exists between AI and scientifi...

**29. [A nice theoretical framework which brings open science to ai4science. However, no concrete steps are proposed to fix ethical issues.](https://openreview.net/forum?id=xm2XYezpLR)**
  **
  > 

**30. [Unlocking the Potential of AI Researchers in Scientific Discovery: What Is Missing?](https://openreview.net/forum?id=z807E7nlcR)**
  *Hengjie Yu, Yaochu Jin*
  📍 CoRR 2025
  > The potential of AI researchers in scientific discovery remains largely untapped. Over the past decade, AI for Science (AI4Science) publications in 145 Nature Index journals have increased fifteen-...

**31. [Artificial Intelligence for Science in Quantum, Atomistic, and Continuum Systems](https://openreview.net/forum?id=9gfQMGh33a)**
  *Xuan Zhang, Limei Wang, Jacob Helwig et al.*
  📍 CoRR 2023
  > Advances in artificial intelligence (AI) are fueling a new paradigm of discoveries in natural sciences. Today, AI has started to advance natural sciences by improving, accelerating, and enabling ou...

**32. [Real-Time Explanations for Tabular Foundation Models](http://arxiv.org/abs/2603.29946v1)**
  *Luan Borges Teodoro Reis Sena, Francisco Galuppo Azevedo*
  > Interpretability is central for scientific machine learning, as understanding \emph{why} models make predictions enables hypothesis generation and validation. While tabular foundation models show s...

**33. [The Recipe Matters More Than the Kitchen:Mathematical Foundations of the AI Weather Prediction Pipeline](http://arxiv.org/abs/2604.01215v1)**
  *Piyush Garg, Diana R. Gergel, Andrew E. Shao et al.*
  > AI weather prediction has advanced rapidly, yet no unified mathematical framework explains what determines forecast skill. Existing theory addresses specific architectural choices rather than the l...

**34. [Super-Resolving Coarse-Resolution Weather Forecasts With Flow Matching](http://arxiv.org/abs/2604.00897v1)**
  *Aymeric Delefosse, Anastase Charantonis, Dominique Béréziat*
  > Machine learning-based weather forecasting models now surpass state-of-the-art numerical weather prediction systems, but training and operating these models at high spatial resolution remains compu...

**35. [Neural Ordinary Differential Equations for Modeling Socio-Economic Dynamics](http://arxiv.org/abs/2604.00632v1)**
  *Sandeep Kumar Samota, Snehashish Chakraverty, Narayan Sethi*
  > Poverty is a complex dynamic challenge that cannot be adequately captured using predefined differential equations. Nowadays, artificial machine learning (ML) methods have demonstrated significant p...

**36. [Representation choice shapes the interpretation of protein conformational dynamics](http://arxiv.org/abs/2604.00580v1)**
  *Axel Giottonini, Thomas Lemmin*
  > Molecular dynamics simulations provide detailed trajectories at the atomic level, but extracting interpretable and robust insights from these high-dimensional data remains challenging. In practice,...

**37. [Activation Saturation and Floquet Spectrum Collapse in Neural ODEs](http://arxiv.org/abs/2604.00543v1)**
  *Nikolaos M. Matzakos*
  > We prove that activation saturation imposes a structural dynamical limitation on autonomous Neural ODEs $\dot{h}=f_θ(h)$ with saturating activations ($\tanh$, sigmoid, etc.): if $q$ hidden layers o...

**38. [Phase space integrity in neural network models of Hamiltonian dynamics: A Lagrangian descriptor approach](http://arxiv.org/abs/2604.00473v1)**
  *Abrari Noor Hasmi, Haralampos Hatzikirou, Hadi Susanto*
  > We propose Lagrangian Descriptors (LDs) as a diagnostic framework for evaluating neural network models of Hamiltonian systems beyond conventional trajectory-based metrics. Standard error measures q...

**39. [Hybrid Energy-Based Models for Physical AI: Provably Stable Identification of Port-Hamiltonian Dynamics](http://arxiv.org/abs/2604.00277v1)**
  *Simone Betteti, Luca Laurenti*
  > Energy-based models (EBMs) implement inference as gradient descent on a learned Lyapunov function, yielding interpretable, structure-preserving alternatives to black-box neural ODEs and aligning na...

**40. [Graph coarsening: From scientific computing to machine learning](https://openreview.net/forum?id=qH2NkVHQRW)**
  *Jie Chen, Yousef Saad, Zechen Zhang*
  > The general method of graph coarsening or graph reduction has been a remarkably useful and ubiquitous tool in scientific computing and it is now just starting to have a similar impact in machine le...

**41. [A Differentiable Programming System to Bridge Machine Learning and Scientific Computing](https://openreview.net/forum?id=u5W9rpksyf)**
  *Mike Innes, Alan Edelman, Keno Fischer et al.*
  📍 CoRR 2019
  > Scientific computing is increasingly incorporating the advancements in machine learning and the ability to work with large amounts of data. At the same time, machine learning models are becoming in...
